e2e/app.e2e.js

```diff
@@ -41,6 +41,22 @@ describe('application launch', function () {
         })
     })
 
+    describe('when removing', function () {
+      it('should remove file', function () {
```

Comment:
Using `should` is not really good specs, it's redundant. http://betterspecs.org/#should